components/dropdown/index.js

```diff
@@ -68,10 +68,15 @@ class Dropdown extends Component {
 			className,
 			contentClassName,
 			expandOnMobile,
+			...additionalProps,
```

Comment:
Why a blanket `additionalProps`, instead of listing the specific `div` props here? Also, you might want to update the component's `README.md`, too.

Reply:
tbh I've seen this pattern used a couple of places and thought to reuse it here e.g. 

https://github.com/WordPress/gutenberg/blob/master/components/icon-button/index.js#L24
https://github.com/WordPress/gutenberg/blob/master/components/placeholder/index.js#L12
https://github.com/WordPress/gutenberg/blob/master/components/button/index.js#L43
https://github.com/WordPress/gutenberg/blob/master/components/external-link/index.js#L18

but definitely no strong feelings about it. I think you are right, I should enumerate the new params. I will update the README too 😃